tools/tests/init_tests.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python3
+
+import sys
+
+from util import *
+
+def init_project(project_name, version, path):
+   out, err = antler_proj_cmd("init -p {0} -n {1} -v {2}".format(path, project_name, version))
+   print(out)
+
+
+def test_init_project():
+   shutil.rmtree("test0")
```

Comment:
File ".../antler-proj/build/tools/tests/init_tests.py", line 41, in <module>
    test_init_project()
  File ".../antler-proj/build/tools/tests/init_tests.py", line 13, in test_init_project
    shutil.rmtree("test0")
  File "/usr/lib/python3.10/shutil.py", line 714, in rmtree
    onerror(os.lstat, path, sys.exc_info())
  File "/usr/lib/python3.10/shutil.py", line 712, in rmtree
    orig_st = os.lstat(path)
FileNotFoundError: [Errno 2] No such file or directory: 'test0'

Reply:
This test was fixed in f17b961